docs/bucket/notifications/README.md

```diff
@@ -602,28 +690,42 @@ func main() {
 }
 ```
 
-```
+Use the following command to execute the program:
+
+```sh
 go run nats.go
+```
+
+A response similar to this one should be displayed:
+
+```
 2016/10/12 06:39:18 Connected
 2016/10/12 06:39:18 Subscribing to subject 'bucketevents'
 ```
 
-Open another terminal and upload a JPEG image into ``images`` bucket.
+Open another console and use the following command to upload a JPEG image into the `images` bucket:
 
-```
+```sh
 mc cp myphoto.jpg myminio/images
 ```
 
-The example ``nats.go`` program prints event notification to console.
+Once the upload completes, return to the first console. A response similar to this one should be displayed:
```

Comment:
Similar feedback as the config get/set, maybe a section called testing at the end?